### PR TITLE
Step3 mark

### PR DESCRIPTION
**Pull request by Maker-Mark**
### Issues

#29 
### Technical Changes
- Implemented basics context API into steps 1 and 2 that allow question state to be passed back and forth.

### Architecture Changes
None

### CODE REVIEW

File: src/landing/Landing.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ import Footer from './Footer';
 import { QuestionItem } from './QuestionItem';
 import { QuestionContext } from '../context';
 import { SafeContainer } from '../common/SafeContainer';
-
+import { Loading } from '../common/Loading'
```
Comment (ethen001) [inline src/landing/Landing.js]: You can simplify your common imports like this:
`import { SafeContainer, Loading } from '../common';`

File: src/questions/Step1.js [hunk 2]
```diff
@@ -25,8 +25,13 @@ export function Step1({ navigation }) {
   // Where we would update the chosen approach state
   useEffect(() => {
     setApproach('Approach 1');
-  }, [ApproachList]);
+  }, []);
```
Comment (ethen001) [inline src/questions/Step1.js]: I don't think this needs to be an effect. You can leave it and we'll discuss this tonight

File: src/questions/Step3.js [hunk 3]
```diff
@@ -13,30 +12,31 @@ import { NextStepButton } from '../common/NextStepButton';
 // eslint-disable-next-line import/named
 import { QuestionContext } from '../context/index';
 import { StepHeader } from '../common';
+import { Loading } from '../common'
```
Comment (ethen001) [inline src/questions/Step3.js]: simplify common imports here as well

File: src/landing/Landing.js (hunk 1 above)
Comment (Maker-Mark) [reply src/landing/Landing.js]: That's right, good call!

**Comment by Maker-Mark** [comment]
> Thank you for your hard work. I love the new Loading component. Please look at my requested changes

Thank you! 
Reviewed and made suggested changes from review. Thank you for the great review!

**Pull request merged**